Workbooks/Azure Monitor - Workspaces/At Scale Insights/At Scale Insights.workbook

```diff
@@ -0,0 +1,453 @@
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 9,
+      "content": {
+        "version": "KqlParameterItem/1.0",
+        "crossComponentResources": [
+          "{Subscription}"
+        ],
+        "parameters": [
+          {
+            "id": "1f74ed9a-e3ed-498d-bd5b-f68f3836a117",
+            "version": "KqlParameterItem/1.0",
+            "name": "Subscription",
+            "label": "Subscriptions",
+            "type": 6,
+            "description": "All subscriptions with Storage accounts",

```

Comment:
these spots in this template refer to storage accounts, and the query filters to only subs with storage accounts?

you need to update those to your resource type